<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>StatisticDialog</class>
 <widget class="QWidget" name="StatisticDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>104</red>
      <green>104</green>
      <blue>104</blue>
     </color>
     <color>
      <red>139</red>
      <green>139</green>
      <blue>139</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>128</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>231</red>
      <green>231</green>
      <blue>231</blue>
     </color>
    </active>
    <inactive>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>104</red>
      <green>104</green>
      <blue>104</blue>
     </color>
     <color>
      <red>139</red>
      <green>139</green>
      <blue>139</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>192</red>
      <green>192</green>
      <blue>192</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>231</red>
      <green>231</green>
      <blue>231</blue>
     </color>
    </inactive>
    <disabled>
     <color>
      <red>104</red>
      <green>104</green>
      <blue>104</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>104</red>
      <green>104</green>
      <blue>104</blue>
     </color>
     <color>
      <red>139</red>
      <green>139</green>
      <blue>139</blue>
     </color>
     <color>
      <red>104</red>
      <green>104</green>
      <blue>104</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>104</red>
      <green>104</green>
      <blue>104</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>128</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>231</red>
      <green>231</green>
      <blue>231</blue>
     </color>
    </disabled>
   </palette>
  </property>
  <property name="font" >
   <font>
    <family>Arial</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget" >
      <column>
       <property name="text" >
        <string>Statistics</string>
       </property>
      </column>
      <item>
       <property name="text" >
        <string>Download</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >:/images/downloads_24x24.png</iconset>
       </property>
       <item>
        <property name="text" >
         <string>Session:</string>
        </property>
        <item>
         <property name="text" >
          <string>Downloaded:</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Average Downloadspeed:</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Count of Downloads:</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text" >
         <string>Overall</string>
        </property>
        <item>
         <property name="text" >
          <string>Downloaded:</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Count of Downloads:</string>
         </property>
        </item>
       </item>
      </item>
      <item>
       <property name="text" >
        <string>Upload</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >:/images/uploads_24x24.png</iconset>
       </property>
       <item>
        <property name="text" >
         <string>Session</string>
        </property>
        <item>
         <property name="text" >
          <string>Uploaded:</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Average Uploadspeed:</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Count of Uploads:</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text" >
         <string>Overall</string>
        </property>
        <item>
         <property name="text" >
          <string>Uploaded</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Count of Uploads:</string>
         </property>
        </item>
       </item>
      </item>
      <item>
       <property name="text" >
        <string>Connections:</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >:/images/connections_24x24.png</iconset>
       </property>
       <item>
        <property name="text" >
         <string>Session</string>
        </property>
        <item>
         <property name="text" >
          <string>Connections:</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Peers:</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text" >
         <string>Overall</string>
        </property>
        <item>
         <property name="text" >
          <string>Connections:</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Peers:</string>
         </property>
        </item>
       </item>
      </item>
      <item>
       <property name="text" >
        <string>Misc</string>
       </property>
       <item>
        <property name="text" >
         <string>Session:</string>
        </property>
        <item>
         <property name="text" >
          <string>Uptime:</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text" >
         <string>Overall:</string>
        </property>
        <item>
         <property name="text" >
          <string>Uptime</string>
         </property>
        </item>
       </item>
      </item>
      <item>
       <property name="text" >
        <string>Records</string>
       </property>
       <item>
        <property name="text" >
         <string>Uploadspeed:</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Downloadspeed:</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Connections:</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Peers:</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Uptime:</string>
        </property>
       </item>
      </item>
     </widget>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="GraphFrame" name="frmGraph" >
         <property name="minimumSize" >
          <size>
           <width>120</width>
           <height>80</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::NoContextMenu</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnToggleSettings" >
           <property name="text" >
            <string>Show Settings</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnReset" >
           <property name="text" >
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frmSettings" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>355</width>
           <height>82</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>355</width>
           <height>82</height>
          </size>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>3</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="chkReceiveRate" >
              <property name="font" >
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="contextMenuPolicy" >
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="toolTip" >
               <string/>
              </property>
              <property name="layoutDirection" >
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text" >
               <string>Receive Rate</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkSendRate" >
              <property name="font" >
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="contextMenuPolicy" >
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="toolTip" >
               <string/>
              </property>
              <property name="layoutDirection" >
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text" >
               <string>Send Rate</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkAlwaysOnTop" >
              <property name="font" >
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="contextMenuPolicy" >
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="layoutDirection" >
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text" >
               <string>Always On Top</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frmOpacity" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>3</number>
             </property>
             <item>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lblPercentOpacity" >
                 <property name="minimumSize" >
                  <size>
                   <width>25</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font" >
                  <font>
                   <family>Arial</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                   <underline>false</underline>
                   <strikeout>false</strikeout>
                  </font>
                 </property>
                 <property name="contextMenuPolicy" >
                  <enum>Qt::NoContextMenu</enum>
                 </property>
                 <property name="layoutDirection" >
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text" >
                  <string>100</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label" >
                 <property name="font" >
                  <font>
                   <family>Arial</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                   <underline>false</underline>
                   <strikeout>false</strikeout>
                  </font>
                 </property>
                 <property name="contextMenuPolicy" >
                  <enum>Qt::NoContextMenu</enum>
                 </property>
                 <property name="text" >
                  <string>% Opaque</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="sldrOpacity" >
               <property name="contextMenuPolicy" >
                <enum>Qt::NoContextMenu</enum>
               </property>
               <property name="toolTip" >
                <string>Changes the transparency of the Bandwidth Graph</string>
               </property>
               <property name="minimum" >
                <number>30</number>
               </property>
               <property name="maximum" >
                <number>100</number>
               </property>
               <property name="value" >
                <number>100</number>
               </property>
               <property name="sliderPosition" >
                <number>100</number>
               </property>
               <property name="tracking" >
                <bool>false</bool>
               </property>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance" >
                <bool>false</bool>
               </property>
               <property name="tickPosition" >
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval" >
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnSaveSettings" >
              <property name="text" >
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancelSettings" >
              <property name="text" >
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>GraphFrame</class>
   <extends>QWidget</extends>
   <header>gui/graphframe.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
